README.md

```diff
@@ -221,9 +236,9 @@ Based on FlightService example.
 import React from 'react';
 import ReactFlow from 'react-flow-renderer';
 import cupid from '@asyncapi/cupid';
-import {getAsyncApiDocuments} from 'utils'; // function for reading AysncAPI files 
+import {getAsyncApiExamples} from 'utils'; // function for reading AsyncAPI files 
```

Comment:
```suggestion
import {getAsyncApiExamples} from './utils'; // function for reading AsyncAPI files 
```

in some Node version (in your case) `utils` indicates global path, located in folder where node is installed, so please change it to the local path.